package.json

```diff
@@ -90,13 +90,14 @@
     "draftail": "^1.2.1",
     "element-closest": "^2.0.2",
     "focus-trap-react": "^3.1.0",
-    "prop-types": "^15.6.0",
-    "react": "^16.2.0",
-    "react-dom": "^16.2.0",
+    "prop-types": "^15.6.2",
+    "react": "^16.4.0",
+    "react-dom": "^16.4.0",
     "react-redux": "^5.0.6",
+    "react-streamfield": "^0.9.2",
     "react-transition-group": "^1.1.3",
-    "redux": "^3.7.2",
-    "redux-thunk": "^2.2.0",
+    "redux": "^4.0.0",
+    "redux-thunk": "^2.3.0",
```

Comment:
Officially (e.g. in the `.nvmrc` file) Wagtail's build tooling runs under node 8. If we can make it compatible with newer versions without too much fiddling, then that's good - otherwise, it would probably make sense to handle that upgrade in a separate PR.